src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Media_Animation/ColorAnimationUsingKeyFrames_Fill.xaml.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace UITests.Windows_UI_Xaml_Media_Animation
 {
-	[Sample("Animations")]
+	[Sample("Animations", IsManualTest = true)]
```

Comment:
Can you still add runtime tests that at least validate that the end result of the animation is the right one? Does this fix fix the animation altogether, or just the fact that the animation was not running but the end color was present?

Reply:
There are preexisting runtime tests for zero-time animations. The problem was that color animations were immediately jumping to the end value instead of interpolating through the duration.